contents/blog/the-posthog-array-1-37-0.mdx

```diff
@@ -0,0 +1,145 @@
+* **Completed a sequence of events:** Find users using your product in a very specific way. For example, "Give me users who added something to their cart and then entered a promo code within a day".
+* **Did not complete a sequence of events:** Find users who aren't using your product as you'd like. For example, "Give me users who added something to their cart but didn't checkout within a day".
+* **Do not have the property:** Find more specific sets of users. For example, "Give me users outside of Europe".
+* **Completed an event for the first time:** Find the newest users of a feature. For example, "Give me users who bought an item for the first time in the last 7 days".
+* **Completed an event regularly:** Find your power users. For example, "Give me users who bought an item in 5 out of the last 7 weeks".
+* **Stopped doing an event:** Find your users that are at risk of churning. For example, "Give me users who haven't bought anything in the last 7 days, but had bought something in the 30 days prior".
+* **Started doing an event again:** Find your users that did not churn. For example, "Give me users who bought something, then didn't for 2 weeks, but have again in the last 7 days".
+
+#### AND/OR operators
+
+In addition to having these new conditions, you can now combine them using complex AND/OR groupings.
+
+For example, you can define a cohort that is "Give me users who are outside of Europe AND who have either regularly bought an item 4 of the last 5 weeks OR bought 3 items in the past week".
+
+With the combination of these new conditions and the new AND/OR operators, you can now define extremely powerful cohorts for finding the exact users that you're looking for.
+
+#### Nested cohorts
+
+But that's not all. You can also use existing cohorts to define new cohorts. For example, you can define a cohort as "Give me users in Cohort X that are not in Cohort Y".
+
+Using these nested cohorts allows you to avoid redefining the same cohort over and over again.
+
+
+### New: Dancing hedgehogs
+
+Our favorite mascot, Max, is starting to make more appearances in the product. Now, you'll see Max dancing as insights load!
+
+<img src={DancingHog} alt="Dancing HedgeHog" style={{ width: 720 }} />
+
+### New: Event and property detail pages
+
+Within data management, a new dedicated detail page exists for all events and properties. These pages allow you to dig into the details. For example, on the event page, you can see:
+* When an event was first and last seen
+* How many times it was sent in the last 30 days
+* The top properties for that events
+* A filterable list of the specific event for you to explore
+
+<img src={EventPage} alt="Event Page" style={{ width: 720 }} />
+
+### Improved: Save event columns
+
+On the live events page, you can now save the custom event columns that you want to see. The columns selected can be saved at the team level, so your colleagues can benefit too.
```

Comment:
`On the live events page, you can now select and reorder the custom event columns that you want to see. The columns selected can be saved at the team level, so your colleagues can benefit too.`